custom/icds_reports/migrations/0073_drop_agg_awc_daily_fn.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.16 on 2018-10-29 14:25
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('icds_reports', '0072_merge_20181029_1425'),
+    ]
+
+    operations = [
+        migrations.RunSQL("DROP FUNCTION aggregate_awc_daily;", "SELECT 1")
```

Comment:
I think that this needs the arguments to the function `aggregate_awc_daily(date)`. not sure if you want to also use `drop function if exists`

Reply:
hmm, must be a change in v10 that supports without the args. Updated d3fedd8